packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,13 +1,36 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import { PaperPlane } from '@strapi/icons';
+
+import { pluginId } from './pluginId';
+
+import type { Plugin } from '@strapi/types';
 
 // eslint-disable-next-line import/no-default-export
-export default {
-  register() {
+const admin: Plugin.Config.AdminInput = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  register(app: any) {
     if (window.strapi.features.isEnabled('cms-content-releases')) {
       // EE Code would be here
       // For testing if the plugin is working, add a console.log
+      app.addMenuLink({
+        to: `/plugins/${pluginId}`,
+        icon: PaperPlane,
+        intlLabel: {
+          id: `${pluginId}.plugin.name`,
+          defaultMessage: 'Releases',
+        },
+        async Component() {
+          const { Releases } = await import(
+            /* webpackChunkName: "content-type-builder" */ './pages/Releases'
```

Comment:
I know we have this "webpackChunkName" in all plugins but I'm not sure what is used for. In any case I guess we should name it "content-releases" or something like that

Reply:
it's used to name the chunk files we generate in the admin build @Feranchz :)